L1Trigger/CSCTriggerPrimitives/src/CSCGEMMotherboard.cc

```diff
@@ -240,8 +253,8 @@ int CSCGEMMotherboard::getRoll(const GEMPadDigiId& p) const { return GEMDetId(p.
 int CSCGEMMotherboard::getRoll(const GEMCoPadDigiId& p) const { return p.second.roll(); }
 
 std::pair<int, int> CSCGEMMotherboard::getRolls(const CSCALCTDigi& alct) const {
-  return std::make_pair((getLUT()->get_csc_wg_to_gem_roll(theParity))[alct.getKeyWG()].first,
-                        (getLUT()->get_csc_wg_to_gem_roll(theParity))[alct.getKeyWG()].second);
+  const auto& mymap(getLUT()->get_csc_wg_to_gem_roll(theParity));
+  return std::make_pair(mymap.at(alct.getKeyWG()).first, mymap.at(alct.getKeyWG()).second);
```

Comment:
isn't this the same as
```suggestion
  return mymap.at(alct.getKeyWG());
```
?